storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java

```diff
@@ -452,10 +452,12 @@ private void notifyAboutPendingChangesIfNotInTransaction() {
                 changesToSend = null;
             }
 
-            if (changesToSend != null) {
+            if (changesToSend != null && changesToSend.size() > 0) {
+                final Set<String> affectedTables = new HashSet<String>();
```

Comment:
let's set some initial value here since on Android collections has 0 capacity by default! for example `2` or `3`